records/scenario/scenario0001.json

```diff
@@ -0,0 +1,242 @@
+[
```

Comment:
From where we got this mock data? Have you verified that, it has no customer references?